### PR TITLE
[WIP] Add new tutorial to service binding guide for binding Quarkus to OpenShift Service Registry

### PR DESCRIPTION
**Pull request by jbyrne-redhat**
**Notes for reviewers**
 
 - Add new conceptual content describing file-based binding for OpenShift Service Registry. [HTML preview](http://file.emea.redhat.com/~jbyrne/j6267/README.html#con-about-service-binding_service-binding).
-  Update `rhoas cluster connect` procedure for Service Registry (in addition to Kafka). Add steps for RBAC. [HTML preview](http://file.emea.redhat.com/~jbyrne/j6267/README.html#proc-connecting-kafka-registry-instance-to-openshift-cluster_service-binding).
-  Add tutorial for binding new Quarkus application to Service Registry (in addition to Kafka). [HTML preview](http://file.emea.redhat.com/~jbyrne/j6267/README.html#con-kafka-registry-binding-quarkus-application-using-cli_service-binding).

### CODE REVIEW

File: service-discovery/README.adoc [hunk 1]
```diff
@@ -726,9 +776,190 @@ When you create the binding connection, the Service Binding Operator injects con
...
+----
+$ oc project my-project
+----
+
+. To create the Quarkus application, deploy a container image provided by {org-name}.
++
+.Deploying an example Quarkus application
+[source,options="nowrap"]
+----
+$ oc new-app quay.io/secondsun/kafka-avro-schema-quickstart
```
Comment (jbyrne-redhat) [inline service-discovery/README.adoc:832]: I realize that this container image does not represent not the finalized application. When we do deliver the finalized app, I'll update this image path and application name as necessary.

File: service-discovery/README.adoc [hunk 2]
```diff
@@ -726,9 +776,190 @@ When you create the binding connection, the Service Binding Operator injects con
...
+
+imagestream.image.openshift.io "kafka-avro-schema-quickstart" created
+deploymentconfig.apps.openshift.io "kafka-avro-schema-quickstart" created
+service "kafka-avro-schema-quickstart" created
+route.route.openshift.io "kafka-avro-schema-quickstart" created
```
Comment (jbyrne-redhat) [inline service-discovery/README.adoc:837]: See https://github.com/redhat-developer/app-services-guides/pull/383/files#r778755633. These names will be reviewed and updated as necessary when we deliver the finalized app.

File: service-discovery/README.adoc [hunk 3]
```diff
@@ -726,9 +776,190 @@ When you create the binding connection, the Service Binding Operator injects con
...
+
+. Get the URL of the route created for the application.
++
+.Getting the route details for the Quarkus application
+[source,options="nowrap"]
+----
+$ oc get route
+
+NAME                            HOST/PORT
+kafka-avro-schema-quickstart    kafka-avro-schema-quickstart-my-project.apps.sandbox-m2.ll9k.p1.openshiftapps.com
```
Comment (jbyrne-redhat) [inline service-discovery/README.adoc:850]: See https://github.com/redhat-developer/app-services-guides/pull/383/files#r778755633. These names will be reviewed and updated as necessary when we deliver the finalized app.

**Comment by jbyrne-redhat** [comment]
@secondsun @wtrocki Just bumping this PR review back up in your queue.

**Review by secondsun — APPROVED** [review]
Gave it a read through, everything seems to be there. Good work! +1

**Comment by wtrocki** [comment]
I was on sick leave. What happened here? Can we announce SBO for service registry?

**Comment by secondsun** [comment]
@wtrocki no

**Comment by secondsun** [comment]
I merged it a bit prematurely. We've reopened the pr.

**Pull request merged**